roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -72,6 +72,16 @@ def get_var(task_vars, *keys, **kwargs):
         raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
     return value
 
+def normalized_release(task_vars):
+    """Helper function to normalize the release format to dotted numbers"""
+    release = str(get_var(task_vars, "openshift_release"))
+    if release.startswith('v'):
+        release = release[1:]  # v3.3 => 3.3
+    return release
+
+def normalized_minor_release(task_vars):
```

Comment:
> Actually I don't like passing around task_vars everywhere to begin with. Why not just pass it to the check constructor to be put in an attribute and used later?

I'm also not in favor of passing `task_vars` everywhere indiscriminately.

Storing `task_vars` in an instance attribute has the potential to make dependencies implicit instead of explicit, what might be more damage than good.

If health checks need to access version information, we could make those read-only properties of OpenShiftCheck.... hmm, I started to sketch some code and realized that `OpenShiftCheck.__init__` doesn't take `task_vars`, only `run` does.

I remember the initial "hope" was that we could load the checks only once per execution, then run it once per host in the inventory. However, in practice the action plugin is executed once per each host... so we could change `OpenShiftCheck` to be a "disposable" instance, to be used only once.